web/auth/auth.go

```diff
@@ -541,21 +541,32 @@ func authorizeForm(c echo.Context) error {
 			"Error":  "Error Invalid scope",
 		})
 	}
+  readOnly := permissions[0].Verbs.ReadOnly()
```

Comment:
We just want to show the permission type *once*, and not once per permission as was the case until now.
I guess it would be better to check all the permissions, but I didn't find a straightforward way to do it? Also, can `permissions` be empty, in which case this line would break things?

Reply:
You can do something like this:

```go
readOnly := true
for _, p := range permissions {
  if !p.Verbs.ReadOnly() {
    readOnly = false
  }
}
```